src/resource/solidDataset.ts

```diff
@@ -1160,11 +1167,9 @@ export async function getWellKnownSolid(
       new URL(urlString).origin
     ).href;
 
-    // Technically, the request here should be public and shouldn't require an
-    // authenticated fetch, however, in some environments, fetcher.ts fails to
-    // load cross-fetch sometimes, which results in this call failing if we
-    // don't pass the fetch method through:
-    return await getSolidDataset(wellKnownSolidUrl, { fetch: options.fetch });
+    return await getSolidDataset(wellKnownSolidUrl, {
```

Comment:
Why is the comment deleted?

Reply:
It no longer applies: `options.fetch` is potentially authenticated, hence the comment explaining why an authenticated fetch could be passed even though it shouldn't be necessary, but now we just pass the default fetch, which we know to not be authenticated.